invenio_rdm_records/services/services.py

```diff
@@ -32,23 +29,25 @@ def __init__(
         config,
         files_service=None,
         draft_files_service=None,
-        secret_links_service=None,
+        access_service=None,
+        grants_service=None,
         pids_service=None,
         review_service=None,
     ):
         """Constructor for RecordService."""
         super().__init__(config, files_service, draft_files_service)
-        self._secret_links = secret_links_service
+        self._access = access_service
         self._pids = pids_service
         self._review = review_service
+        self._grants = grants_service
```

Comment:
We miss a 
```python
@property
def grants(self):
   """Record grants service."""
    return self._grants
```
? Or I think that's not used anymore? Probably after you merged everything to access_service?

Reply:
Good catch, that's a leftover!